src/index.html

```diff
@@ -1,14 +1,19 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Stop watch</title>
-  <link rel="stylesheet" href="./styles/main.scss">
-</head>
-<body>
-  <h1>Stop watch</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Stop watch</title>
+    <link rel="stylesheet" href="./styles/main.scss">
+  </head>
+  <body class="container">
+    <div class="clockWrapper">
+      <div class="clockWrapper__cercle">
+        <div class="clockWrapper__minute"></div>
+        <div class="clockWrapper__hour"></div>
+      </div>
+    </div>
```

Comment:
```suggestion
    <div class="clock">
      <div class="clock__minute"></div>
      <div class="clock__hour"></div>
      <div class="clock__circle"></div>
    </div>
```

Strange BEM structure, propose you a better